src/components/editorElements/titleArea/view/titleAreaView.js

```diff
@@ -63,7 +63,7 @@ export default class TitleAreaView extends Component {
             this.setState({ height: event.nativeEvent.contentSize.height });
           }}
           autoFocus={autoFocus}
-          onChangeText={text => this._handleOnChange(text)}
+          onChangeText={textT => this._handleOnChange(textT)}
```

Comment:
what is T meanin

Reply:
It is just making variable different than already used variable name

Reply:
Linting complained so I tried to decrease warnings and slowly we can get rid of all warnings from linting